apigentools/utils.py

```diff
@@ -361,7 +364,13 @@ def write_full_spec(spec_dir, spec_version, spec_sections, fs_path):
                 full_spec[k] = loaded[k]
 
     with open(fs_path, "w", encoding="utf-8") as f:
-        yaml.dump(full_spec, f, Dumper=CSafeDumper)
+        f.write(
+            yaml.dump(
+                full_spec,
+                Dumper=yamlordereddictloader.SafeDumper,
+                default_flow_style=False,
+            )
+        )
```

Comment:
This might have performance impact on very large spec files. Can you try to use:

```suggestion
        yaml.dump(
            full_spec,
            f,
            Dumper=yamlordereddictloader.SafeDumper,
            default_flow_style=False,
        )
```